Archiver.py

```diff
@@ -4,15 +4,19 @@
 
 def read_file(file_path):
     file_path = os.path.abspath(file_path)
-    file_name = os.path.basename(file_path).split(".")[0]
+    file_name = os.path.basename(file_path)[:-4]
```

Comment:
Just some to think about in future (not related to current PR)
I can't see the file extension check in the project. So it's a good idea to add it in future versions.
Also can't say the [-4] check looks good for me. It's nice that you are using slicing, but it will not work in case you'll have file with .docx extension, f.e. (since you are not checking extensions at all)
And the requirements say only that the program is working with .txt files, but this doesn't mean the user will not try to compress f.e. .docx file. And in case he does, we, obviously, need to show corresponding error msg

Reply:
Rewrote this function, now it only works with the "txt" file. Please review the changes.